.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
 .idea
 .env
-src/key.json
-conversion/public/
\ No newline at end of file
+api/key.json
```

Comment:
Tried moving this to the project root, but doesn't look like it'll work without specific configuration for the serverless function as we discussed on slack. Will have to have another go at it